projects/spectator/tsconfig.spec.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../out-tsc/spec",
-    "target": "es5",
+    "target": "es2015",
```

Comment:
For whatever reason running with `target` set to `es5` made it so that none of the template bindings were working with jest.

Changing this to `es2015` seems to have resolved the problem.